src/common/components/repository-row/index.js

```diff
@@ -10,11 +10,13 @@ import { Message } from '../forms';
 import templateYaml from './template-yaml.js';
 
 import { signIntoStore } from '../../actions/auth-store';
-import { registerName } from '../../actions/register-name';
+import { registerName, registerNameError } from '../../actions/register-name';
 import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
 import styles from './repositoryRow.css';
 
+const MINIMUM_SNAP_NAME_LENGTH = 1;
```

Comment:
Yeah, I actually added this thinking it was more than 1, but I checked with @fgallina and it's just 1 ... I could remove this but it's maybe useful for the future.